packages/graphiql-react/src/ui/markdown.css

```diff
@@ -6,6 +6,8 @@
  *   (field description any optionally deprecation reason).
```

Comment:
ok just leaving an idea here, could be done in a different PR - what do you think of CSS, design primitives and other non-react-specific elements going in the `@graphiql/toolkit` directory?

Reply:
Opened a discussion here to align on this (I don't think this is actionable yet): https://github.com/graphql/graphiql/discussions/2585